packages/docs/src/stories/docs/general/04-FAQ.mdx

```diff
@@ -174,7 +174,7 @@ import '../../../../../components/src/components/accordion-group/accordion-group
           How can I contribute to the Solid Design System?
         </h4>
         <span class="sd-prose">
-          We welcome contributions from the community as the Solid Design System is an open-source project. To get started, please review our <a href="https://solid-design-system.fe.union-investment.de/docs/?path=/docs/docs-general-contributing--docs#contributing-to-solid-components">Contributing Guide</a> carefully. Additionally, make sure to familiarize yourself with our <a target="_blank" href="https://github.com/solid-design-system/solid?tab=coc-ov-file">Code of Conduct</a> and our <a target="_blank" href="https://github.com/solid-design-system/solid/blob/main/DEVELOPER_STARTING_GUIDE.md">Developer Starting Guide</a>. These resources will provide you with the necessary information and guidelines for contributing effectively to our project.
+          We welcome contributions from the community as the Solid Design System is an open-source project. To get started, please review our <a href="https://solid-design-system.fe.union-investment.de/docs/?path=/docs/docs-general-contributing--docs#contributing-to-solid-components">Contributing Guide</a> carefully. Additionally, make sure to familiarize yourself with our <a target="_blank" href="https://github.com/solid-design-system/solid?tab=coc-ov-file">Code of Conduct</a> and our <a target="_blank" href="https://github.com/solid-design-system/solid/blob/main/README.md">Developer Starting Guide</a>. These resources will provide you with the necessary information and guidelines for contributing effectively to our project.
```

Comment:
why did you change this URL? In case you really want to point to the readme, the link title should (IMO) be changed as well.

Reply:
The `developer starting guide` doesn't exist anymore. The `readme` file contains some of the information that was in the previous file so we either link to it or just remove this part.

Reply:
```suggestion
          We welcome contributions from the community as the Solid Design System is an open-source project. To get started, please review our <a href="https://solid-design-system.fe.union-investment.de/docs/?path=/docs/docs-general-contributing--docs#contributing-to-solid-components">Contributing Guide</a> carefully. Additionally, make sure to familiarize yourself with our <a target="_blank" href="https://github.com/solid-design-system/solid?tab=coc-ov-file">Code of Conduct</a> and our <a target="_blank" href="https://github.com/solid-design-system/solid/blob/main/README.md">Readme</a>. These resources will provide you with the necessary information and guidelines for contributing effectively to our project.
```

Reply:
ah. I did not know that.